dashboards/Prometheus.json

```diff
@@ -3647,14 +3647,11 @@
                     "steppedLine": false,
                     "targets": [
                         {
-                            "expr": "time() - node_time_seconds",
-                            "format": "time_series",
+                            "expr": "clamp_min((time() - max by (node_name) (node_time_seconds)) - scalar(round(prometheus_target_interval_length_seconds{quantile=\"0.99\",interval=~\"$scrapingInterval\"})),0)",
```

Comment:
The question remains: why round to the integer at all? why not to one or two digits after dot? If I understand correctly, that will make a graph a bit more smooth.

Reply:
Let's simplify it by removing a rounding function completely.